fornax.py

```diff
@@ -245,18 +245,19 @@ def process_data_info(self):
                     # TO ACCESS REGION-RESTRICTED DATA ANONYMOUSLY.
                     # -----------------------
                     # Attempting anonymous access:
-                    msg = 'Accessing region data anonymously ...'
-                    s3_config = botocore.client.Config(signature_version=botocore.UNSIGNED)
-                    s3_resource = boto3.resource(service_name='s3', config=s3_config)
-                    accessible, message = self.is_accessible(s3_resource, aws_info['bucket'], aws_info['path'])
-                    if accessible:
-                        break
-                    message = f'  - {msg} {message}.'
-                    messages.append(message)
+                    if data_access == 'region': 
```

Comment:
should there be a case for `'restrictred'`, too, or we don't need to raise an error/warning for that?

Reply:
Anonymous access is attempted only for the 'region' case. It wouldn't make sense to try it for 'restricted' mode. Though, I don't think that it is even possible with 'region' mode, so this whole part will likely be removed. I am still waiting for a confirmation from the experts.